tests/browser/contacts_accounts/merge_contacts.robot

```diff
@@ -0,0 +1,44 @@
+*** Settings ***
+
+Resource        tests/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Merge Contacts
+    # Manually creating contact records, sosl doesn't always return api generated contacts
+    Go To Object Home  Contact
+    Click Link  title:New
+    Populate Form
+    ...                       First Name=Robot
+    ...                       Last Name=One
+    Click Modal Button        Save & New
+    Populate Form
+    ...                       First Name=Robot
+    ...                       Last Name=Two
+    Click Modal Button        Save
+    ${winning_id} =  Get Current Record Id
+    Store Session Record  Contact  ${winning_id}
+    Go To Object Home  Contact
+    Page Should Contain  Robot One
+    Page Should Contain  Robot Two
+    Go To Setup Home
+    Open App Launcher
+    Input Text  //input[@placeholder='Search apps or items...']  Contact Merge
+    Click Link  Contact Merge
+    Select Frame with ID  vfFrameId
+    Input Text  //input[@placeholder='Search Contacts']  Robot
+    Click Element  //input[@value='Search']
+    Click Element  (//span[@id='fauxCBSelect'])[1]
+    Click Element  (//span[@id='fauxCBSelect'])[2]
+    Click Element  //input[@value='Next']
```

Comment:
This suggestion is optional. In case u want to keep the robot file clean, u can create a generic locator for checkbox and add it to locator_w19.py file and create a keyword for it in NPSP.py file. Here is the generic locator '//tbody/tr[./td[2]/a[contains(text(),"{}")]]/td//input[@type="checkbox"]'. To your keyword you would pass the contact name in order to select checkbox next to it. If you are not comfortable I can clean this up later.

Reply:
@skristem that locator doesn't look very generic; it looks very tied to the exact DOM structure of the page. This way is at least targeting by id, which is less likely to break. I agree it would be ideal to locate based on the actual contact name rather than position though.

Reply:
@davisagli i agree but if there is another contact created by earlier robot test that had robot in the name and was not deleted, then there is a possibility of merging wrong contacts.